### ISSUE TITLE
uiowa.dev.drupal.uiowa.edu CSS 

### ISSUE THREAD

**Issue by inspector508** [uiowa/uids#70]
class "banner__summary" fails color contrast: **rule specifies same value for color and background**. If secondary background (image, color, background, etc) fails to load, the content will not be visible.

Understand this is not real content, just acknowledging a CSS issue that may go away on its own, but just in case...
